graphene/contrib/django/filter/fields.py

```diff
@@ -26,7 +27,13 @@ def get_queryset(self, qs, args, info):
         filter_kwargs = self.get_filter_kwargs(args)
         order = self.get_order(args)
         if order:
-            qs = qs.order_by(order)
+            qs = qs.order_by(to_snake_case(order))
```

Comment:
should `get_order` which is the function designed to handle the API level (`args`) to the `to_snake_case` so it would "own" the API layer and let `get_queryset` deal with `qs` level things? (separation of concerns / layering).